src/components/Navigation.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+
+const Navigation = () => {
+  return (
+    <div className="Navigation">
+      <ul>
+        <NavLink
+          to={"/"}
+          className={(nav) => (nav.isActive ? "nav-active" : "")}
+        >
+          <li>Accueil</li>
```

Comment:
Declare an array to list URL names like `["Accueil", "A Propos"]` and use `.map()` function and one `li` to render them. 
This way you can dynamically add or remove a link, without the need to duplicate the `li`